apps/prairielearn/src/pages/instructorAssessmentAccessOverrides/instructorAssessmentAccessOverrides.ejs

```diff
@@ -0,0 +1,321 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <%- include('../partials/head'); %>
+</head>
+<body>
+  <script>
+    function toDatetimeLocal(date) {
+        var year = date.getFullYear();
+        var month = ('0' + (date.getMonth() + 1)).slice(-2);
+        var day = ('0' + date.getDate()).slice(-2);
+        var hours = ('0' + date.getHours()).slice(-2);
+        var minutes = ('0' + date.getMinutes()).slice(-2);
+        return year + '-' + month + '-' + day + 'T' + hours + ':' + minutes;
+    }
+
+   
+    $(function () {
+      $('[data-toggle="popover"]').popover({ sanitize: false,   container: 'body' });
+
+      $('#add_new_override').on('submit', function (event) {
+        var startDate = new Date($('#start_date').val());
+        var endDate = new Date($('#end_date').val());
+      
+        if (startDate >= endDate) {
+          event.preventDefault();
+          $('#end_date').addClass('is-invalid');
+          $('#end_date_error').text('End date should be greater than the start date.').show();
+        }
+        
+        
+      });
+      $('.edit-override-button').on('click', function (event) {
+        event.preventDefault();
+        var button = $(this);
+        var editOverrideForm = $('#edit-override-form');
+        editOverrideForm.find('#edit-assessment_id').val(button.data('assessment-id'));
+        editOverrideForm.find('#edit-student_uid').val(button.data('user-id'));
+        editOverrideForm.find('#edit-group_id').val(button.data('group-id'));
+        editOverrideForm.find('#edit-credit').val(button.data('credit'));
+        var policy_start_date = new Date(button.data('start-date'));     
+        var policy_start_date_string = toDatetimeLocal(policy_start_date);
+        var policy_end_date = new Date(button.data('end-date'));
+        var policy_end_date_string = toDatetimeLocal(policy_end_date);
+
+        editOverrideForm.find('#edit-start_date').val(policy_start_date_string);
+        editOverrideForm.find('#edit-end_date').val(policy_end_date_string);
+        editOverrideForm.find('#edit-created_by').val(button.data('created-by'));
+        editOverrideForm.find('#edit-note').val(button.data('note'));
+        editOverrideForm.find('#edit-type').val(button.data('type'));
+        var policyId = $(this).data('policy-id');
+        $('#edit-override-form input[name="policy_id"]').val(policyId);	
+        var groupName = $(this).data('group-name');
+        $('#edit-group_name').val(groupName);	
+        $('#editPolicyModal').modal('show'); // Show the modal form
```

Comment:
This comment doesn't add much that isn't already clear from the code.

```suggestion
        $('#editPolicyModal').modal('show');
```

Reply:
done in [33e0fde](https://github.com/PrairieLearn/PrairieLearn/pull/9103/commits/33e0fde889ceaa842183714c28d771b6412d24d0)